docs/experimental-features.md

```diff
@@ -365,3 +365,30 @@ ipfs config --json Experimental.ShardingEnabled true
 
 - [ ] Make sure that objects that don't have to be sharded aren't
 - [ ] Generalize sharding and define a new layer between IPLD and IPFS
+
+---
+
+## IPNS pubsub
+
+### In Version
+
+0.4.13
+
+### State
+
+Experimental, default-disabled.
+
+Utilizes pubsub for publishing ipns records in real time.
```

Comment:
Can we say slightly more about how it works/what it does? e.g:

> When you publish an IPNS record, a message about the record will also be published via pubsub (note: does it get repeatedly sent [and roughly what frequency?] or just once at publish time?). When you resolve an IPNS address, your daemon will subscribe to the pubsub topic for that name so it is immediately informed of changes.

I’m not even totally sure my understanding there is correct, so please fix if it’s not!

Reply:
Really, the important part is: what should the user expect and why would they want to try this feature out?

Reply:
Yes. Exactly that ^

Reply:
ok fine, I'll add _another_ sentence :)